README.md

```diff
@@ -16,7 +16,7 @@ Refer to the [Go SDK's developer documentation](https://docs.developers.optimize
   
 ### Requirements  
 
-Requires Golang version 1.19 or higher.
+Requires Golang version 1.21.0 or higher.
```

Comment:
Should we have a minimum of 1.20.7 (latest release of still supported version)?

Reply:
We can leave 1.19 for now. When we release the go-sdk with go 1.21, then this must be updated to 1.21.